docs/user/integration/sap-cloud-logging/README.md

```diff
@@ -1,21 +1,48 @@
 # Integrate with SAP Cloud Logging
 
-SAP Cloud Logging is an instance-based and environment-agnostic observability service that builds upon OpenSearch to store, visualize, and analyze logs, metrics, and traces. This guide explains how to define LogPipelines and TracePipelines to ingest application and access logs as well as distributed trace data in instances of SAP Cloud Logging.
+## Overview
+
+| Category| |
+| - | - |
+| Signal types | logs, traces, metrics |
+| Backend type | third-party remote |
+| OTLP-native | yes |
+
+Learn how to define LogPipelines and TracePipelines to ingest application and access logs as well as distributed trace data in instances of [SAP Cloud Logging](https://help.sap.com/docs/cloud-logging?locale=en-US&version=Cloud).
+
+SAP Cloud Logging is an instance-based and environment-agnostic observability service that builds upon OpenSearch to store, visualize, and analyze logs, metrics, and traces.
+
+![setup](./../assets/sap-cloud-logging.drawio.svg)
+
+## Table of Content
+
+- [Integrate with SAP Cloud Logging](#integrate-with-sap-cloud-logging)
+  - [Overview](#overview)
+  - [Table of Content](#table-of-content)
+  - [Prerequisites](#prerequisites)
+  - [Ship Logs to SAP Cloud Logging](#ship-logs-to-sap-cloud-logging)
+  - [Ship Distributed Traces to SAP Cloud Logging](#ship-distributed-traces-to-sap-cloud-logging)
+  - [Ship Metrics to SAP Cloud Logging (experimental)](#ship-metrics-to-sap-cloud-logging-experimental)
+  - [Kyma Dashboard Integration](#kyma-dashboard-integration)
 
 ## Prerequisites
 
-- An instance of SAP Cloud Logging with OpenTelemetry enabled to ingest distributed traces
-- A Secret named `sap-cloud-logging` in the `sap-cloud-logging-integration` namespace, holding the credentials and endpoints for the instance
+- Kyma as the target deployment environment
+- The [Telemetry module](../README.md) is [enabled](https://kyma-project.io/#/02-get-started/08-install-uninstall-upgrade-kyma-module?id=install-uninstall-and-upgrade-kyma-with-a-module)
+- An instance of SAP Cloud Logging with OpenTelemetry enabled to ingest distributed traces.
+  >**TIP:** It's recommended to create it with the SAP BTP Service Operator (see [Create an SAP Cloud Logging Instance through SAP BTP Service Operator](https://help.sap.com/docs/cloud-logging/cloud-logging/create-sap-cloud-logging-instance-through-sap-btp-service-operator?locale=en-US&version=Cloud)), because it takes care of creation and rotation of the required Secret. However, you can choose any other method of creating the instance and the Secret, as long as you make sure that OTLP ingestion is enabled [see Configuration Parameters](https://help.sap.com/docs/cloud-logging/cloud-logging/configuration-parameters?locale=en-US&version=Cloud) in the instance.
```

Comment:
I'd place "see Configuration Parameters" in brackets or make it a separate sentence

Reply:
You're right, much clearer! 

```suggestion
  >**TIP:** It's recommended to create it with the SAP BTP Service Operator (see [Create an SAP Cloud Logging Instance through SAP BTP Service Operator](https://help.sap.com/docs/cloud-logging/cloud-logging/create-sap-cloud-logging-instance-through-sap-btp-service-operator?locale=en-US&version=Cloud)), because it takes care of creation and rotation of the required Secret. However, you can choose any other method of creating the instance and the Secret, as long as you make sure that OTLP ingestion is enabled in the instance. For details, see [Configuration Parameters](https://help.sap.com/docs/cloud-logging/cloud-logging/configuration-parameters?locale=en-US&version=Cloud).
```